test/fixtures/fake_item.json

```diff
@@ -0,0 +1,18 @@
+{
+    "id": "item_JDcbIdX9xojCje",
+    "active": true,
+    "name": "English Augusts",
+    "description": "An indian story, Prize winner.",
+    "amount": 20000,
+    "unit_amount": 20000,
+    "currency": "INR",
+    "type": "invoice",
+    "unit": null,
+    "tax_inclusive": false,
+    "hsn_code": null,
+    "sac_code": null,
+    "tax_rate": null,
+    "tax_id": null,
+    "tax_group_id": null,
+    "created_at": 1647584637
+  }
```

Comment:
Representation of item entity not according to doc - https://razorpay.com/docs/api/items/#create-an-item

Reply:
Done